source/Projectile.js

```diff
@@ -0,0 +1,23 @@
+var Pixi = require("pixi.js")
+var Reference = require("Reference.js")
+
+export default class Projectile extends Pixi.Sprite {
+    constructor(x, y) {
+        var projectileTex =
+            Pixi.Texture.fromImage(require("./images/projectile.png"))
```

Comment:
So I'm feeling like we should move the textures out of the class; otherwise, we're loading, parsing, and instantiating a separate texture for each and every projectile, which was one thing for the Junkership (where we'll have, like, four of those, each with a unique texture), but is an entirely other thing for Projectiles (where we'll have dozens on the screen at once, all with the same texture.)